lib/auth/httpAuthMiddleware.js

```diff
@@ -7,23 +8,64 @@ const { Strategy } = require('./strategy')
 let options
 let passport
 let httpNodeApp
+let client
+const httpTokenCache = {}
 
 module.exports = {
     init (_options) {
         options = _options
-        return (req, res, next) => {
-            try {
-                if (req.session.ffSession) {
-                    next()
-                } else {
-                    req.session.redirectTo = req.originalUrl
-                    passport.authenticate('FlowFuse', { session: false })(req, res, next)
+        return [
+            async (req, res, next) => {
+                try {
+                    if (req.session.ffSession) {
+                        next()
+                    } else if (req.get('Authorization')?.startsWith('Bearer')) {
+                        // need to make a HTTP request against the forge platform to check
+                        // the token. Should we cache the token to reduce round trip on every
+                        // request?
+                        // We should include the Project ID and the path along with the token
+                        // to be checked
+                        const token = req.get('Authorization').split(' ')[1]
+                        const cacheHit = httpTokenCache[token] 
```

Comment:
```suggestion
                        const cacheHit = httpTokenCache[token]
```
Trailing spaces not allowed.eslint[no-trailing-spaces](https://eslint.org/docs/latest/rules/no-trailing-spaces)